### PR TITLE
update port log and health status

### PR DESCRIPTION
**Pull request by baixinsui**

### CODE REVIEW

File: log/log.go [hunk 1]
```diff
@@ -33,12 +34,12 @@ func InitLog(level, path string) error {
 		basic.SetFormatter(&logrus.JSONFormatter{})
 	} else {
 		basic.Formatter = &logrus.TextFormatter{
-			TimestampFormat: "2023/01/01 - 01:01:01",
+			TimestampFormat: time.DateTime,
```
Comment (iskey) [inline log/log.go:37]:
```suggestion
			TimestampFormat: "2006/01/02 - 15:04:05",
```

File: log/log.go [hunk 2]
```diff
@@ -33,12 +34,12 @@ func InitLog(level, path string) error {
...
 			FullTimestamp:   true,
 		}
 
 		basic.Formatter = &logrus.TextFormatter{
-			TimestampFormat: "2023/01/01 - 01:01:01",
+			TimestampFormat: time.DateTime,
```
Comment (iskey) [inline log/log.go:42]:
```suggestion
			TimestampFormat: "2006/01/02 - 15:04:05",
```

**Review by iskey — CHANGES_REQUESTED** [review]
comments added.
The TimestampFormat should be a fixed timestamp.
https://stackoverflow.com/questions/42217308/go-time-format-how-to-understand-meaning-of-2006-01-02-layout

File: server/health_status.go [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package server
+
+type HealthStatus struct {
+	HealthStatus string `json:"healthStatus" binding:"required"`
+}
+
+const OK = "OK"
```
Comment (iskey) [inline server/health_status.go:7]: You don't need to use variables here, `string` is ok.
or a new Enum would be better if you insist.

File: server/health_status.go [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package server
```
Comment (iskey) [inline server/health_status.go:1]: License should be added here.

**Review by iskey — CHANGES_REQUESTED** [review]
some minor comments added.

File: config/conf.go [hunk 5]
```diff
@@ -43,7 +43,7 @@ type SectionSSL struct {
 var defaultConf = []byte(`
 mode: release
 host: "localhost" # ip address to bind (default: any)
-port: "8080" # ignore this port number if auto_tls is enabled (listen 443).
+port: "9091" # ignore this port number if auto_tls is enabled (listen 443).
```
Comment (swaroopar) [inline config/conf.go:46]: in case of auto_tls, we can use there 9443 instead of the default port.

File: server/health_status.go [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+
+type HealthStatus struct {
```
Comment (swaroopar) [inline server/health_status.go:3]: would it make sense to move all types to a different directory?

File: server/health_status.go [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+	HealthStatus string `json:"healthStatus" binding:"required"`
```
Comment (swaroopar) [inline server/health_status.go:4]: yes enum is better here.

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: server/handle_policy_eval.go [hunk 8]
```diff
@@ -42,9 +42,7 @@ func abortWithError(c *gin.Context, code int, message string) {
 }
 
 func healthHandler(c *gin.Context) {
-	c.JSON(200, gin.H{
-		"healthStatus": "OK",
-	})
+	c.JSON(200, HealthStatus{"OK"})
 	return
```
Comment (swaroopar) [inline server/handle_policy_eval.go:46]: do we need this explicit `return`?

File: .gitignore [hunk 9]
```diff
@@ -0,0 +1,6 @@
+# Default ignored files
+.vscode
+.idea
+policy-man.iml
```
Comment (swaroopar) [inline .gitignore:4]: its better to ignore all *.iml files.

**Review by swaroopar — CHANGES_REQUESTED** [review]

**Review by iskey — APPROVED** [review]
LGTM

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**